cmake/3rdParty/Platform/Linux/BuiltInPackages_linux.cmake

```diff
@@ -47,3 +46,7 @@ ly_associate_package(PACKAGE_NAME lz4-1.9.3-vcpkg-rev4-linux
 ly_associate_package(PACKAGE_NAME pyside2-5.15.2-rev2-linux                         TARGETS pyside2                     PACKAGE_HASH 7589c397c8224d0c3ad691ff02e1afd55d9a1f9de1967c14eb8105dd2b0c4dd1)
 ly_associate_package(PACKAGE_NAME SQLite-3.37.2-rev1-linux                          TARGETS SQLite                      PACKAGE_HASH bee80d6c6db3e312c1f4f089c90894436ea9c9b74d67256d8c1fb00d4d81fe46)
 ly_associate_package(PACKAGE_NAME AwsIotDeviceSdkCpp-1.15.2-rev1-linux              TARGETS AwsIotDeviceSdkCpp          PACKAGE_HASH 83fc1711404d3e5b2faabb1134e97cc92b748d8b87ff4ea99599d8c750b8eff0)
+
+# Use system default OpenSSL library instead of maintaining an O3DE version for Linux
+include(${CMAKE_CURRENT_LIST_DIR}/OpenSSL_linux.cmake)
+
```

Comment:
It doesnt feel right to include it here since this is not a built-in package

Reply:
Perhaps a better way is to follow the same pattern as 'VkValidation' as an example of a platform specific 3rd party.

Reply:
I can cut another GHI to rename the BuiltInPackages_xxxxx files and make corresponding changes in other cmake files that reference these BuiltInPackages_xxxxx files currently if we agree with the solution. Updating the comment in the current file makes sense to me.